integration/images/include/datadog/demo_env.rb

```diff
@@ -1,23 +1,24 @@
 module Datadog
```

Comment:
Touching this file to change a few runtime references triggered Rubocop's full fury, thus a few extra lint changes on a separate commit are present here.